cluster/gce/gci/configure-helper.sh

```diff
@@ -2686,6 +2681,16 @@ oom_score = -999
 [plugins.cri.registry.mirrors."docker.io"]
   endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]
 EOF
+
+  # Reuse docker group for containerd.
+  local containerd_gid="$(cat /etc/group | grep ^docker: | cut -d: -f 3)"
+  if [[ ! -z "${containerd_gid:-}" ]]; then
+    cat >> "${config_path}" <<EOF
+# reuse id of the docker group
+[grpc]
+  gid = ${containerd_gid}
+EOF
+  fi
```

Comment:
do we need an else? if docker is not around, what gid is used?

Reply:
@liggitt we let containerd default to whatever it needs to. 

https://github.com/containerd/containerd/blame/master/docs/man/containerd-config.toml.5.md#L45-L49